tests/fixtures/formats/early_return_outside_main.nu

```diff
@@ -0,0 +1,11 @@
+let y = 'not hiya'
+
+return
+
+def test [] {
+    error make {msg: "this should not show"}
+}
```

Comment:
is that relevant to the test?

i could check the fix without that part, so i'm just wondering if i'm missing something :relieved:

Reply:
Oh, I forget to remove `test` function, thanks!